src/i18n/languages/english.js

```diff
@@ -31,7 +31,10 @@ const upload = {
 	dragFilesHere: 'Drag files here',
 	chooseFiles: 'Choose files',
 	uploadTextDisabledMessage: 'Some files were uploaded. You might either upload file or paste text.',
-	uploadTextPlaceholder: 'Paste text here..'
+	uploadTextPlaceholder: 'Paste text here..',
+	emailPolicyTitle: '<h1 class="text-2xl font-bold">Conditions for the processing of personal data</h1>',
```

Comment:
Please, do not add the html code to the translations. What if we wanted to changer the html tag or css classes? We would have to make it in both places.. What if we had thousands of language translations?